.github/buildomat/jobs/tuf-repo.sh

```diff
@@ -5,7 +5,10 @@
 #: target = "helios-latest"
 #: output_rules = [
 #:	"=/work/manifest.toml",
-#:	"=/work/repo.zip*",
+#:	"=/work/repo-dogfood.zip*",
```

Comment:
Should there also be a `[publish]` section? The predictable locations are nice

Reply:
I think I was waiting on Buildomat allowing uploads larger than 1 GiB so that we don't need to change downstream tools to stop fetching the parta/partb files, but I can add that now if we want.

Reply:
fair, that's fine then. there's still plenty of manual steps and this already streamlines a lot

Reply:
... realizing there already was `[[publish]]` for these, but I didn't update them. fixing that, oops.